### ISSUE TITLE
Accessibility - Most Issues Will Aplly for the Website

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#44]
Most of these issues will apply to any necessary website updates too.

**Comment by adriennerlong** [comment]
Have you been able to test the website platform and identify additional issues? If the website is not ready to launch on September 19, we will do a later release. Our main priority for the September 19 launch is the IOS app.

**Comment by jfc3-dol** [comment]
@adriennerlong,

I have not seen the ComplyChain as a website any where.

Do you have a link/URL to the website I can test?

Most likely the same issues will be found for the website as for the iOS application. There might be a few different ones but should be the same if you are creating it from the same code base.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Start testing the website after finishing my testing of the iOS and Android versions.

**Comment by adriennerlong** [comment]
From ICF: Fixed.  All changes are mirrored on all platforms.

**Comment by jfc3-dol** [comment]
Going to close this issue because we tagged and the issues that apply to more than one platform using different labels.
